.cloud-build/ExecuteChangedNotebooks.py

```diff
@@ -84,41 +180,68 @@ def run_changed_notebooks(
     artifacts_path.joinpath("success").mkdir(parents=True, exist_ok=True)
     artifacts_path.joinpath("failure").mkdir(parents=True, exist_ok=True)
 
-    passed_notebooks: List[str] = []
-    failed_notebooks: List[str] = []
+    notebook_execution_results: List[NotebookExecutionResult] = []
 
     if len(notebooks) > 0:
         print(f"Found {len(notebooks)} modified notebooks: {notebooks}")
 
-        for notebook in notebooks:
-            print(f"Running notebook: {notebook}")
-
-            # TODO: Handle cases where multiple notebooks have the same name
-            try:
-                ExecuteNotebook.execute_notebook(
-                    notebook_file_path=notebook,
-                    output_file_folder=artifacts_path,
-                    replacement_map={
-                        "PROJECT_ID": variable_project_id,
-                        "REGION": variable_region,
-                    },
+        if should_parallelize and len(notebooks) > 1:
+            print(
+                "Running notebooks in parallel, so no logs will be displayed. Please wait..."
+            )
+            with concurrent.futures.ThreadPoolExecutor(max_workers=None) as executor:
+                notebook_execution_results = list(
+                    executor.map(
+                        functools.partial(
+                            execute_notebook,
+                            artifacts_path,
+                            variable_project_id,
+                            variable_region,
+                            False,
```

Comment:
Don't show logs when running notebooks in parallel due to interleaved logs from multiple notebooks.